ansible_files/roles/equinix/heterogeneous_create_infra/defaults/main.yaml

```diff
@@ -0,0 +1,9 @@
+primary_device_group_name: "primary"
+secondary_device_group_name: "secondary"
+equinix_ssh_user: "root"
+equinix_plans: "{{
+      {
+       primary_device_group_name : 'c3.small.x86',
+       secondary_device_group_name : 'c3.small.x86'
```

Comment:
The plans are not definitive, it's just they are cheap. I'll change them with another once the full playbook is ready.